apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateCsvWritersFromDslMojoIT/expected/BeneficiaryCsvWriter.java

```diff
@@ -0,0 +1,24 @@
+package gov.cms.test;
+
+import java.lang.Object;
+import java.lang.String;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public final class BeneficiaryCsvWriter {
```

Comment:
Missing various docs in this class (Checkstyle should be catching this...)

Reply:
This isn't a compiled class.  It's an expected output from the code generator.  If checkstyle was configured to test it that would be a bug.